src/cljc/rems/common/application_util.cljc

```diff
@@ -27,3 +29,25 @@
 
 (defn is-handler? [application user]
   (contains? (workflow-handlers application) user))
+
+(defn- str-to-int [s]
+  #?(:clj (Integer/parseInt s)
+     :cljs (js/parseInt s 10)))
+
+(defn- printToConsole [s]
```

Comment:
naming-convention-is-kebab-case also this function is not required?

Reply:
yes, I will delete it.